invenio_theme/templates/semantic-ui/invenio_theme/header.html

```diff
@@ -0,0 +1,83 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2015-2018 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+
+<header>
+
+  {%- block navbar %}
+  <nav class="ui inverted menu">
+    <div class="ui container">
+
+      {%- block navbar_header %}
+      {%- block brand %}
+      {%- if config.THEME_LOGO %}
+      <a href="/" class="header item"><img class="logo" src="{{ url_for('static', filename=config.THEME_LOGO)}}"
+          alt="{{_(config.THEME_SITENAME)}}" /></a>
+      {%- else %}
+      <a class="logo" href="/">{{_(config.THEME_SITENAME)}}</a>
+      {%- endif %}
+      {%- endblock %}
+      {%- endblock navbar_header %}
+
+      {%- block navbar_inner %}
+
+      {%- block navbar_nav %}
+      {%- if config.THEME_SEARCHBAR %}
+      {%- block navbar_search %}
+
+
+      <div class="ui basic segment">
+        <form class="ui form" action="{{config.THEME_SEARCH_ENDPOINT}}" role="search">
```

Comment:
can please have an html ID here please? for react portals :)

Reply:
Actually shouldn't you override the `navbar_search` block instead in `invenio-search-ui`? This is how it was done before with Angular: https://github.com/inveniosoftware/invenio-search-ui/blob/2ebb88809f7da8f79c043980c9fbc73c493e05e5/invenio_search_ui/templates/invenio_search_ui/header.html#L12-L17

Reply:
we could avoid overwriting, with this one simple trick (config var in the theme):
https://github.com/inveniosoftware/invenio-search-ui/pull/77/files#diff-3df67d0fb49b645e7423d1875da3b6c2R110